cfgs/agent/alm.yaml

```diff
@@ -1,28 +1,27 @@
 #common
 agent: 'alm'
 device: 'cuda'
-seed: 640
+seed: 639
 
 #benchmark
 benchmark: 'gym'
 id: 'HalfCheetah-v2'
 
 #algorithmic config
-rollout: true_model # alm, rollout_correction, 1_step_correction, true_model
+rollout: rollout_correction # alm, rollout_correction, 1_step_correction
```

Comment:
Please, use alm for this experiment, I want to see only changes from replacing the critic